### ISSUE TITLE
could not find function "combineSQM"

### ISSUE THREAD

**Issue by jahid10** [jtamames/SqueezeMeta#572]
Hi,
I have completed my analysis with Squeezemeta.pl in coassembly mode. Now I want to analyze the data with R. But when I ran,
>data = ("/media/bioinfo/hdd1/Jahid_NGS_metagenomics/soil_sample_data
")
  combineSQM (
  bacteria, archaea,
  tax_source = "orfs",
  trusted_functions_only = F,
  ignore_unclassified_functions = F,
  rescale_tpm = T,
  rescale_copy_number = T
)
Error in combineSQM(bacteria, archaea, tax_source = "orfs", trusted_functions_only = F,  : 
  could not find function "combineSQM"

**Comment by fpusan** [comment]
Hi!
Is this happening in windows or Mac?
Is the package properly installed?
What do you get when running `library(SQMtools)`?

**Comment by jahid10** [comment]
Hi, this is in Linux. I properly installed SqueezeMeta in conda environment using the installation guide. I can activate R in termnial activating the SqueezeMeta envrionment. However, I cannot install SQMtools in R studio (v4.20+) which is already installed in my linux and use the functions of SQMtools.

**Comment by fpusan** [comment]
Ah, then it makes sense. RStudio does not have access to any of the libraries in your conda environment.
You'll have to install the package manually.
For this:
1) Activate the environment
2) Locate the environment with `echo $CONDA_PREFIX`
3) Inside RStudio, type `install.packages("/path/to/envs/SqueezeMeta/SqueezeMeta/SQMtools", repos = NULL, type = "source")`.
4) This will ask for several dependencies, which you can install normally in RStudio.
Let me know if this works!

**Comment by jahid10** [comment]
Hi,
Thanks! the solve worked very well. However, I have another question not pertaining to this issue. I used R command 
> plotTaxonomy(Hadza, rank='family
![myfigure](https://user-images.githubusercontent.com/65849689/200472083-d11f63c7-3e04-4d63-a59d-5c6dc9df365a.svg)
', count='percent') 
to plot the different phylum in my samples. However I got a large percentage identified as 'unmapped'.  I didnt use the --euk filter and my sample may contain large amount of host genome (_Chorcorus_ spp.). Is there a possibility that these unmapped taxa belong to eukaryotes and something else?
